packages/web/src/components/tipping/support/TopSupporters.tsx

```diff
@@ -38,7 +37,7 @@ export const TopSupporters = () => {
     : {}
   const rankedSupporters = useSelector<AppState, User[]>(state => {
     const usersMap = getUsers(state, {
-      ids: Object.keys(supportersForProfile).map(k => parseInt(k))
+      ids: Object.keys(supportersForProfile).map(k => (k as unknown) as ID)
```

Comment:
are we not calling parseInt now? in that case we could just do `Object.keys(...) as unknown as ID[]` ?